spec/menu_spec.rb

```diff
@@ -0,0 +1,7 @@
+require './lib/menu.rb'
+describe Menu do
+  let(:menu) { double :menu }
```

Comment:
1. Redundant double; doesn't get used?
2. Also, no point in creating a double of the described class in its own test file.